examples/README.md

```diff
@@ -138,3 +146,33 @@ Set up a Hermes relayer between two chains.
 * **Inventory file:** [`inventory-hermes.yml`](inventory-hermes.yml)
 
 Follow the [Hermes IBC Relayer Setup](/docs/Hermes-Relayer-Setup.md) guide in the `docs` folder for all the requirements and steps needed to deploy the relayer. 
+
+## Run a Big Dipper block explorer
+
+Run a block explorer for your chain using the Big Dipper [software](https://bigdipper.live/) ([docs](https://docs.bigdipper.live/)) software.
+
+* **Inventory file:** [`inventory-bigdipper.yml`](inventory-bigdipper.yml)
+
+1. Install ansible-galaxy dependencies: `ansible-galaxy install -r requirements.yml`.
+2. Set up a node. For example:
+    ```
+    ansible-playbook node.yml -i examples/inventory-theta.yml -e 'target=SERVER_DOMAIN'
+    ```
+3. Go and edit [`inventory-bigdipper.yml`](inventory-bigdipper.yml) to your liking. Some things like `hasura_admin_secret` and `letsencrypt_email` will need to be changed, while others are optional.
+4. Setup DNS:
+    ```
+    mydomain.com.        3600 IN A     123.123.123.123
+    hasura.mydomain.com. 3600 IN CNAME mydomain.com.
+    rpc.mydomain.com.    3600 IN CNAME mydomain.com.
+    ```
+    You can configure the subdomain names if needed using other variables:
+    ```yaml
+    # Defaults: 
+    hasura_host: "hasura."
+    rpc_host: "rpc."
+    bdui_host: ""
+    ```
+5. Install Big Dipper:
+    ```
+    ansible-playbook bigdipper.yml -i examples/inventory-bigdipper.yml -e 'target=SERVER_DOMAIN'
```

Comment:
I ran this line with the inventory file as-is and it stopped here:
```
TASK [bigdipper : copy genesis.json] *******************************************************************************************************************************************************************************
fatal: [{{ target }}]: FAILED! => {"changed": false, "msg": "Source /home/gaia/.gaia/config/genesis.json not found"}
```

Reply:
I've specified that for this example, the blockchain node and Big Dipper installation need to be on the same machine.